nutrition_labels/grant_tagger.py

```diff
@@ -172,9 +180,12 @@ def fit_transform(self, data, train_data_id="Internal ID"):
             print("Vectorizer type not recognised")
         self.X_vect = self.vectorizer.fit_transform(self.X)
 
+        # For BERT/SciBERT + naive bayes the values need to be between 0 and 1, 
+        # so scale the values.
+        self.scaler = None
         if "bert" in self.vectorizer_type and self.classifier_type == "naive_bayes":
-            scaler = MinMaxScaler()
-            self.X_vect = scaler.fit_transform(self.X_vect)
+            self.scaler = MinMaxScaler()
```

Comment:
you can add that to be part of the vectorizer if you define the vectorizer as a pipeline. pipelines in sklearn are composable i.e. one pipeline can contain multiple pipelines and your vectorizer can be a pipeline

Reply:
good idea - will add this when I correct https://github.com/wellcometrust/nutrition-labels/issues/72